.github/workflows/build.yaml

```diff
@@ -254,10 +254,10 @@ jobs:
         env:
           DOCKER_REGISTRY: ${{ env.CONTAINER_REGISTRY }}
           DOCKER_TAG_VERSION: latest
-      - name: Push container images (PR)
+      - name: Build container images (PR) # Don't push on PR, agent will not have permission.
```

Comment:
if we never push on PR, how will we run functional tests using the PR changes?

Reply:
The functional tests workflow already does a build/push of the containers it uses.